gabarit/template_nlp/nlp_project/package_name/models_training/models_tensorflow/utils_deep_keras.py

```diff
@@ -323,7 +323,8 @@ def get_config(self) -> Any:
         return config
 
     def build(self, input_shape) -> None:
-        assert len(input_shape) == 3
+        if len(input_shape) != 3:
+            raise ValueError("input_shape should be 3")
```

Comment:
Should we add a "raises" section in the class docstring ? Or do we wait for a refacto of the experimental parts ?

Reply:
I think we should wait for the refactoring of the experimental parts